Sources/Datadog/DatadogCore/DatadogCore.swift

```diff
@@ -206,9 +206,6 @@ internal final class DatadogCore {
         allUploads.forEach { $0.flushAndTearDown() }
         allStorages.forEach { $0.setIgnoreFilesAgeWhenReading(to: false) }
 
-        // Deinitialize V2 Integrations (arbitrarily for now, until we make it into `DatadogFeatureIntegration`):
-        get(feature: CrashReporter.self)?.deinitialize()
-
```

Comment:
Doesn't this imply more cleanup possibilities? Notably:
* `var v1Features`
* `extension DatadogCore: DatadogV1CoreProtocol`
* `internal protocol V1Feature`